src/XMakeTasks/Telemetry.cs

```diff
@@ -0,0 +1,55 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.Build.Framework;
+using System.Collections;
+
+namespace Microsoft.Build.Tasks
+{
+    public sealed class Telemetry : TaskExtension
```

Comment:
Any recommendations for the name of this task?

Logging a message is just `<Message />`, logging an error is just `<Error />`, so logging telemetry is just `<Telemetry />`.